src/ui.h

```diff
@@ -172,6 +172,7 @@ enum class Command : uint32_t {
     // Help
     LOCALE,
     WEBSITE,
+    GIHUB,
```

Comment:
Why GIHUB without the T?

Reply:
Misguided attempt to avoid potential name conflicts? I'll change it.